qemu/tests/cfg/nested_test.cfg

```diff
@@ -2,6 +2,7 @@
     kar_repo = <kar_repo_url>
     cert_url = <cert_ca_url>
     nested_bs_options = ""
+    machine_type = ""
```

Comment:
`machine_type` is a param that already be used by L1, need use different param name for L2
such as `l2_machine_type`

Reply:
Fixed.